src/container/Container.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState } from 'react';
+import styles from './container.module.scss';
+import Header from '../components/Header/Header';
+import SearchBar from '../components/SearchBar/SearchBar';
+import Content from '../components/Content/Content';
+import { TMovieCard } from '../types/TypeMovieDB';
+import {
+  getMoviePopuler,
+  getMovies,
+  getMovieTrend,
+  getMovieYears,
+  searchMovies,
+} from '../services/movie';
+
+const Container = () => {
+  const [movies, setMovies] = useState<TMovieCard[]>([]);
+  const [loading, setLoading] = useState<boolean>(false);
+  const [searchTerm, setSearchTerm] = useState<string>('');
+  const [error, setError] = useState<string | null>(null);
+  const [noResults, setNoResults] = useState<string>('');
+
+  const searchMovie = async (e: { preventDefault: () => void }) => {
+    e.preventDefault();
+    if (searchTerm === '') {
+      return alert('Please enter a movie name');
+    } else {
+      await searchMovies(searchTerm).then((res) => {
+        setMovies(res.results);
+        setSearchTerm('');
+      });
+    }
+  };
+  useEffect(() => {
+    setLoading(true);
+    setTimeout(() => {
+      getMovies()
+        .then((res) => {
+          setLoading(false);
+          setMovies(res.results);
+        })
+        .catch((err) => {
+          setLoading(false);
+          setError(err.message);
+          setNoResults('No results found');
```

Comment:
no results aslında `res.results.length` in 0 olduğu durum değil mi?

Reply:
Evet hocam bunu hata blogundan mı çıkaralım

Reply:
O zaman evet. Çünkü diğer türlü error'a düşmez ve hiç bi zaman yakalayamazsın.

Reply:
istenilen değişiklikler düzenlendi.